tribomobile/components/SideBarOption.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react';
+import { TouchableHighlight, Text, View, TouchableOpacity } from 'react-native';
+
+const  BarGray = () => {
+  return(
+    <View style={{width: 15}}>
+      <Text> </Text>
+  </View>
+  );
+}
+
+const SideBarOption = (props)=>{
+  const [hide, setHide] = useState(false);
+
+  const {
+    textSideBar,
+    textSideBar2
```

Comment:
textSideBar2 can have a different name. It is important to name our variables/props/state with a self-descriptive name in order to be understandable by all the team easily